controllers/operatorconfigmap_controller.go

```diff
@@ -194,80 +229,55 @@ func (c *ClusterVersionReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			"config.json": "[]",
 		},
 	}
-	if err := c.own(monConfigMap); err != nil {
+	if err := c.own(encConfigMap, operatorConfigMap); err != nil {
 		return ctrl.Result{}, err
 	}
-	err = c.create(encConfigMap)
-	if err != nil && !k8serrors.IsAlreadyExists(err) {
+	if err := c.create(encConfigMap); err != nil && !k8serrors.IsAlreadyExists(err) {
 		c.log.Error(err, "failed to create monitor configmap", "name", encConfigMap.Name)
 		return ctrl.Result{}, err
 	}
 
-	c.cephFSDeployment = &appsv1.Deployment{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      csi.CephFSDeploymentName,
-			Namespace: c.OperatorNamespace,
-		},
-	}
```

Comment:
as discussed, removals of this initialization is the reason for panic that I mentioned earlier, pls reinstate and test.